sdk/couchbase-core/src/analyticsx/analytics.rs

```diff
@@ -72,17 +72,45 @@ impl<C: Client> Analytics<C> {
         self.http_client.execute(req).await
     }
 
-    pub async fn query<'a>(&self, opts: &QueryOptions<'a>) -> error::Result<QueryRespReader> {
-        let client_context_id = opts.client_context_id.map(|c| c.to_string());
-        let statement = opts.statement;
-        let on_behalf_of = opts.on_behalf_of;
+    pub async fn query<'a>(&self, opts: &QueryOptions) -> error::Result<QueryRespReader> {
```

Comment:
Same here, is the lifetime specifier still needed?

Reply:
Nope, same as the other one